src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,92 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import React from 'react';
+
+type Props = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (pageNum: number) => void,
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange = () => {},
```

Comment:
Why do you need a default value here if the onPageChange function is required in a Props type?
```suggestion
  onPageChange,
```